base/challenge-skeleton/README.md

```diff
@@ -1,52 +1,61 @@
+* Edit `challenge/image/Dockerfile` to change the commandline or the files you want to include.
 * Try the challenge locally with `make test-docker`.
-* If you have a cluster ready, deploy the challenge with `make start`.
-  * To access it, create a port forward with `make port-forward` and connect via `nc localhost PORT` using the printed port.
+* If you have prepared a cluster, deploy the challenge with `make start`.
+  * To access the challenge, create a port forward with `make port-forward` and connect to it via `nc localhost PORT` using the printed port.
 * Check out `make <tab>` for more commands.
 
-## healthcheck
+## Directory layout
 
-* The healthcheck needs to run a webserver on port 45281 that responds to /healthz requests.
- * 200 if the challenge is healthy, otherwise an error (e.g. 400).
- * You can find an example webserver in `healthcheck/image/healthz.py`.
-* The base image comes preloaded with pwntools.
-* If your exploit is written in python, just drop it in `healthcheck/image/doit.py`.
+The following directories are available:
+
+### /config
 
-# Directory layout:
+The `config` directory holds a few configuration files:
 
-## /config
+* `chal.conf`: Configure this file if you are deploying the challenge to the cluster, if it's publicly accessible, and/or if it has a healthcheck.
+* `pow.conf`: Contains the difficulty of the proof-of-work, 0 means disabled.
+* `advanced`: The Kubernetes config used to deploy the challenge. E.g. you can add a tmpfs here or change the port.
 
-The config directory holds a few configuration files:
+### /challenge
 
-* chal.conf: Configure if the challenge should be deployed to the cluster, if it's publicly accessible and if it has a healthcheck.
-* pow.conf: Contains the difficulty of the proof-of-work, 0 means disabled.
-* advanced: The kubernetes config used to deploy the challenge. E.g. you can add a tmpfs here or change the port.
+The `challenge` directory contains the challenge binary and anything else related to the challenge:
 
-## /challenge
+* `image/`: Dockerfile and files of the challenge image.
+* `Makefile`: Edit this file if you need to run code before building the Docker image, e.g. for building the challenge from source.
 
-The challenge binary and anything else challenge related goes in here.
+### /healthcheck
 
-* image/: Dockerfile and files of the challenge image.
-* Makefile: You can edit this file if you need to run code before building the docker image, e.g. for building the challenge from source.
+The `healthcheck` directory is optional. If you don't want to write a healthcheck, feel free to delete the directory. However, we strongly recommend that you implement a healthcheck :).
 
-## /healthcheck
+* `image/`: Dockerfile and files of the healthcheck.
+* `Makefile`: Edit this file if you need to build any custom files before running your healthcheck.
```

Comment:
Same as before, maybe instead of "before running" it should say "when building".

Reply:
Done